### ISSUE TITLE
Voice not Corresponding with the Text

### ISSUE THREAD

**Issue by mfdsrax2** [07th-mod/guide#22]
I just started playing Onikakushi (JP version) and downloaded Voice and Graphics Patch here. But found that some voices are not linked with the corresponding text. The voice for the marked text appears in a later dialog.

I played this game for less than half an hour and found quite a lot of voices unsyncing. I believe that must be a big issue. Will someone look into this?
![20180304214441_1](https://user-images.githubusercontent.com/13324717/36959570-9c49cef4-1ff7-11e8-907e-28220e03ee17.jpg)
![20180304214732_1](https://user-images.githubusercontent.com/13324717/36959571-9c600070-1ff7-11e8-9263-ff8180e61b5f.jpg)

**Comment by ItaloKnox** [comment]
I would normally close down the issue for being in the wrong place, but since you provided enough information I will accept it.

Anyway, I checked the script and you're right. I'm really unsure how this happened, but there is a clear mismatch for a lot of voices in this scene. I will be fixing them soon. It would be really helpful if you could keep this updated with more. If the game turns to a point where it's all synced again, please tell me where this happens.

**Comment by mfdsrax2** [comment]
Thanks for the quick reply. I will update it as I play the game.

By the way, where is a proper place for this issue?

**Comment by ItaloKnox** [comment]
Yes, in the [Onikakushi repository](https://github.com/07th-mod/onikakushi/issues/). This repository is for guide issues.

By the way, do you remember how many issues you have found? Is it more than those you posted:

**Comment by mfdsrax2** [comment]
I remember there are issues with some dialog after the last screenshot. I don't have my laptop with me right now but I'll keep updating them.

**Comment by ItaloKnox** [comment]
[This commit](https://github.com/07th-mod/onikakushi/commit/4239aba0e430224fa4aed879d68c4a1e81f8a87c) fix the voices I found that was in the wrong place. Looks like there was a small confusion with the position of the dialog, I'm surprised we didn't find out about this since we went through the game in Japanese at least once while adding PS3 content.

In any case, if you find more please report them to me. You can keep this issue open or make a new one in the Onikakushi repo, it's fine either way since I already opened an exception.

**Comment by enumag** [comment]
@ItaloKnox Wrong branch for such fixes...
